localstack/testing/aws/lambda_utils.py

```diff
@@ -109,7 +108,19 @@ def package_for_lang(scenario: str, runtime: str, root_folder: Path) -> str:
         return package_path
 
     # packaging
-    result = subprocess.run(["make", "build"], cwd=runtime_dir)
```

Comment:
[L90 above](https://github.com/localstack/localstack/pull/10441/files#diff-51f9f58239b7e55074f6f55c5c019aeec4f16cffede7c7afb8bf7b91f636dcf5R90): We could consider renaming `common` to multiruntime to make the reuse a bit more explicit.